src/main/charts/confluence/templates/statefulset-synchrony.yaml

```diff
@@ -74,4 +75,5 @@ spec:
           configMap:
             name: {{ include "synchrony.fullname" . }}-entrypoint
             defaultMode: 0744
+        {{ include "confluence.volumes.sharedHome" . | nindent 8 }}
```

Comment:
I believe this is baking in an assumption that your libraries are located in the `shared-home` volume. Currently, if customers want to use a different volume (e.g. they will create a libraryVolume just for jar files), then they cannot use it in synchrony.

In Confluence stateful set we are using additional volumes - https://github.com/atlassian-labs/data-center-helm-charts/blob/master/src/main/charts/confluence/templates/_helpers.tpl#L237

This allows you to define `confluence.additionalLibraries` but if you are using different than local/shared home persistent volume, you also need to add the volume definition to `volumes.additional`. It is probably not the most elegant, but it is possible. For synchrony with this change, it won't be possible as `volumes.additional` is not mounted to synchrony statefulSet.

I think we can either add new additional volumes for synchrony or also add the `volumes.additional` the same way as it is currently done for Confluence. I would prefer just mounting the same volumes as it is a niche use case (ideally you use shared-home volume).

```
{{- with .Values.volumes.additional }}
{{- toYaml . | nindent 0 }}
{{- end }}
```

Please read through https://github.com/atlassian-labs/data-center-helm-charts/blob/master/CONFIG.md#additional-libraries--plugins especially:
> If you're not using the shared-home volume, then you can declare your own custom volume in the "Additional Volumes" section below, then declare the libraries as above (but with your custom volume name).